### ISSUE TITLE
make sources clearer for screen readers

### ISSUE THREAD

**Issue by AaronONS** [open-sdg/open-sdg#2032]
Brock to estimate.

DAC Usability issue
Users reported that the purpose and structure of information was not clear in places.

Screen reader users noted that they were unable to identify which content was a ‘Source’ within the ‘Sources’ tabpanel.

Screen reader user comments: “While navigating through the source information in the chart I was unable to clearly define which information referred to a source . This is problematic as I am unable to accurately define the information. I would find it helpful if the information were made clearer, so I am able to understand and get what I need from it.”

URL
https://sdgdata.gov.uk/5-1-1/

Solution:
Consider including a heading, similarly to that used for indicator 5.5.1, to introduce the table.

Example:
<h2>Source 1</h2>

Note: Additional information can be found in: Accessibility Audit Report for ONS SDG, 21/04/2021, ref-000699 p.54

**Comment by sarah-beasley** [comment]
Make it so it's just visible for screen readers - doesnt need to say 'Source 1' for everyone else.
